src/Pages/Explorar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Footer from '../components/Footer';
+
+function Explorar() {
+  return (
+    <div className="">
```

Comment:
O que acha de remover esse className já que ele esta vazio? 🤔 
Vou deixar a mesma sugetão para outros que eu vi logo abaixo caso faça sentido para você é só aceitar. 😄 
```suggestion
    <div>
```